.github/workflows/code-coverage.yml

```diff
@@ -26,6 +26,19 @@ jobs:
       with:
         fetch-depth: 0 # fetching all
 
+    - uses: actions/setup-dotnet@v1
```

Comment:
we don't need to install 3.1.x/5.0.x because they are already installed in windows-latest. I always check this: 
- https://github.com/actions/virtual-environments
- https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md

Remember that `windows-latest` can change in the future to windows-2022, so, always have that in mind.